egs/cifar/v1/image/get_image2num_frames.py

```diff
@@ -14,7 +14,7 @@
 import os
 import sys
 import numpy as np
-from scipy import misc
+from PIL import Image
```

Comment:
I'm a little surprised to see you replace scipy with PIL.  I though we were going to move, in the long term at least, from PIL to scikit-image, at least I see a conversation in my email about this.  @hhadian might want to comment.

Also, I won't let this block merging this PR, but we should add a script to check python library dependencies.
One options is to have a script image/check_dependencies.sh and have it check dependencies for all the things used in the image/ scripts-- we can add options to enable or disable certain checks as needed depending on where it's called form. @hhadian, what do you think?

Reply:
sorry, In one of the online forms, it was mentioned that we can find the size of the image without loading it into memory (https://stackoverflow.com/questions/15800704/get-image-size-without-loading-image-into-memory). Since in the get_image2num_frames.py script, it was obtaining size of images, I updated it after discussing with @hhadian. It was saving around 100 mins for madcat arabic data. 

get_image2num_frames with PIL is taking around 70 mins
get_image2num_frames with scipy is taking around 168 mins

Reply:
I was planning to add parallelization, I was not able to get a chance. 
I will try imageio and add parallelizartion

Reply:
sorry, I tried skimage for create_line_image_from_page_image.py, but I was getting an error which i was not able to fix. so i shifted back to PIL there also. I will fix it in both places.

Reply:
It is in .tif format. I was able to do all the processing required in create_line_image_from_page_image. I was able to view the processed line images in my local machine. But while saving, the image it was saving a completely black image after rounding off the values which were close to zero.

Reply:
I tried to find an option in save function in skimage which will fix it but i was not able to find a particular option.